rpc_client.go

```diff
@@ -118,12 +118,7 @@ func (client QuobyteClient) sendRequest(method string, request interface{}, resp
 		if resp.StatusCode == 401 {
 			return errors.New("Unable to authenticate with Quobyte API service")
 		}
-		body, err := ioutil.ReadAll(resp.Body)
-		if err != nil {
-			return (err)
-		}
-		log.Printf("Warning: Dumping full reply body:\n%s\n", string(body))
-		return errors.New("JsonRPC failed, see plugin logfile for details")
+		return fmt.Errorf("JsonRPC failed with error code %d", resp.StatusCode)
```

Comment:
Could you also append the body as part of the error. We need body to determine the cause of error.

The body does not contain any sensitive data but the cause for failure. It does not have to be part of the log but must be communicated back to the user.

Reply:
@misterikkit I have reverted the fix as error code alone is not sufficient to determine the cause of failure. Please add the body of response to error and repost it. Thanks!